ci/docker/base.Dockerfile

```diff
@@ -24,13 +24,22 @@ RUN apt-get update -qq && apt-get install -qq -y --no-install-recommends \
     libffi-dev \
     libhdf5-dev \
     liblzma-dev \
-    python-openssl \
+    python3-openssl \
     libreadline-dev \
     git \
-    rustc \
+    jq \
     htop && \
     rm -rf /var/lib/apt/lists/*
 
+# Install Rust using rustup
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+RUN rustc --version && which rustc && cargo --version && which cargo
+
+# Install Bencher for performance monitoring
+RUN cargo install --git https://github.com/bencherdev/bencher --branch main --locked --force bencher_cli
+RUN bencher --version
+
```

Comment:
Why do we install from sources instead of the binaries? https://bencher.dev/docs/how-to/install-cli/#linux-mac--unix or https://bencher.dev/docs/how-to/install-cli/#packages

Reply:
I tried but the binaries do not support our architecture.

Reply:
For me the ARM64 binaries work on alps.

Reply:
In the CI it did not work though

Reply:
I will re-try it ;) (just to be sure)

Reply:
Indeed, it works. I commit it now.